filebeat/module/shorewall/log/test/default.log

```diff
@@ -0,0 +1 @@
+May 28 17:23:25 myHost kernel: [3124658.791874] Shorewall:FORWARD:REJECT:IN=eth2 OUT=eth2 SRC=1.2.3.4 DST=1.2.3.4 LEN=141 TOS=0x00 PREC=0x00 TTL=63 ID=55251 PROTO=UDP SPT=5353 DPT=5353 LEN=121
```

Comment:
It will be nice to have a handful more example lines with different configurations to avoid "happy paths". I guess that this line is the default logging configuration but Filebeat must be able to handle also missing fields. For example, I don't know if it's possible that `OUT=eth2` appears as `OUT=` if no interface is defined as _out_ or it will simply dissapear from the line.

Reply:
Added!